credmark/cmf/types/ledger.py

```diff
@@ -482,7 +483,7 @@ def __call__(self,  # pylint: disable=too-many-arguments
                         limit='5')
                 # ret.data contains a list of row dicts, keyed by column name
             """
-            context = credmark.cmf.model.ModelContext.current_context()
+            context = self.context
```

Comment:
I'm not 100% sure this is correct if a contract is returned from one local model to another. It would be better to use the current_context().

Reply:
reverted.